phpstan.neon.dist

```diff
@@ -95,3 +95,8 @@ parameters:
             message: '~Method Doctrine\\DBAL\\Driver\\PDOSqlsrv\\Connection\:\:lastInsertId\(\) should return string but returns string\|false\|null\.~'
             paths:
                 - %currentWorkingDirectory%/lib/Doctrine/DBAL/Driver/PDOSqlsrv/Connection.php
+
+        -
+            message: '~Method Doctrine\\DBAL\\Portability\\Connection::prepare\(\) should return Doctrine\\DBAL\\Statement but returns Doctrine\\DBAL\\Portability\\Statement\.~'
```

Comment:
This is caused by the issue earlier identified in https://github.com/doctrine/dbal/pull/4045:

> The portability layer is currently implemented on two levels of abstraction at the same time: most of its components implement driver-level interfaces but the `Connection` class extends the DBAL connection.